docs/user/dashboard/lens.asciidoc

```diff
@@ -197,11 +199,15 @@ Annotations allow you to call out specific points in your visualizations that ar
 [role="screenshot"]
 image::images/lens_annotations_8.2.0.png[Lens annotations]
 
+Create the annotation. 
```

Comment:
Is this line necessary?  It looks a little strange placed after the image.  Or maybe make it bold without the ending period.

Reply:
I had an idea to break this all up, but then I got distracted and forgot to remove this rogue line. Let me work some magic here.